ibc/packet.go

```diff
@@ -0,0 +1,82 @@
+package ibc
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	host "github.com/cosmos/ibc-go/v3/modules/core/24-host"
+)
+
+// Packet is a packet sent over an IBC channel as defined in ICS-4.
+// See: https://github.com/cosmos/ibc/blob/master/spec/core/ics-004-channel-and-packet-semantics/README.md
```

Comment:
Consider using a link to a tag or a specific commit hash, in case the file is ever moved in the future.